app/src/main/java/org/mozilla/focus/topsites/TopSitesOverlay.kt

```diff
@@ -44,7 +44,7 @@ fun TopSitesOverlay(modifier: Modifier = Modifier) {
                 topSites = topSites,
                 onTopSiteClicked = { item ->
                     Shortcuts.shortcutOpenedCounter.add()
-
+                    components.settings.shouldMergeTopSiteTab = true
```

Comment:
From what I remember I tried that but didn't work, also if only the url is loaded I'm not sure if the three dots menu is updated. I will check again

Reply:
If you're able to, it would be interesting to know if this removing the `MergeTabsMiddleware` also solves the problem. It could be that this middleware is causing conflicts with the tab logic (which was intentional at the time of the experiment).

Reply:
It didn't work because I was using the wrong tab id 🤦